submitted_models/costar_shafter_sensor_config_1/model.sdf

```diff
@@ -0,0 +1,550 @@
+                <camera name="realsense_front">
+                    <horizontal_fov>1.18682</horizontal_fov>
+                    <lens>
+                        <intrinsics>
+                            <fx>617.61480</fx>
+                            <fy>617.61480</fy>
+                            <cx>321.1996</cx>
+                            <cy>234.8967</cy>
+                            <s>0</s>
+                        </intrinsics>
+                    </lens>
+                    <distortion>
+                        <k1>0.0</k1>
+                        <k2>0.0</k2>
+                        <k3>0.0</k3>
+                        <p1>0.0</p1>
+                        <p2>0.0</p2>
+                    </distortion>
+                    <image>
+                        <width>640</width>
+                        <height>480</height>
+                        <format>R8G8B8</format>
+                    </image>
+                    <clip>
+                        <near>0.01</near>
+                        <far>300</far>
+                    </clip>
+                    <depth_camera>
+                      <clip>
+                        <near>0.1</near>
+                        <far>10</far>
+                      </clip>
+                    </depth_camera>
+                    <noise>
+                        <type>gaussian</type>
+                        <mean>0</mean>
+                        <stddev>0.007</stddev>
+                    </noise>
+                </camera>
+            </sensor>
+
+            <visual name="laser_visual_base_1">
+                <pose>0.01 0 0.0772 0 0 0</pose>
+                <geometry>
+                    <mesh>
+            	        <scale>1 1 1</scale>
+            	        <uri>meshes/VLP16_base_1.dae</uri>
+             	    </mesh>
+                </geometry>
+            </visual>
+ 	    <visual name="laser_visual_base_2">
+                <pose>0.01 0 0.0772 0 0 0</pose>
+                <geometry>
+                    <mesh>
+            	        <scale>1 1 1</scale>
+            	        <uri>meshes/VLP16_base_2.dae</uri>
+          	    </mesh>
+                </geometry>
+            </visual>
+            <visual name="laser_visual_scan">
+                <pose>0.01 0 0.0772 0 0 0</pose>
+                <geometry>
+                    <mesh>
+            	        <scale>1 1 1</scale>
+            	        <uri>meshes/VLP16_scan.dae</uri>
+          	    </mesh>
+                </geometry>
+            </visual>
+            <sensor name="front_laser" type="gpu_ray">
+                <pose>0 0 0.12 0 0 0</pose>
+                <update_rate>10</update_rate>
+                <lidar>
+                    <scan>
+                        <horizontal>
+                            <!-- Resolution can't be above 16k, see
+                            https://bitbucket.org/ignitionrobotics/ign-sensors/issues/8 -->
+                            <samples>10000</samples>
```

Comment:
This should be 1800 at 10Hz

http://www.mapix.com/wp-content/uploads/2018/07/63-9229_Rev-H_Puck-_Datasheet_Web-1.pdf

Reply:
Good point thank you. The part has been updated.

Reply:
While we are at it. It would be better to update Husky configs as well (May be in a seperate PR) https://github.com/osrf/subt/blob/master/submitted_models/costar_husky_sensor_config_1/model.sdf#L291 https://github.com/osrf/subt/blob/master/submitted_models/costar_husky_sensor_config_2/model.sdf#L291